fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/Operator.java

```diff
@@ -53,6 +53,11 @@ public abstract class Operator {
     // or self reference of groups
     protected long salt = 0;
 
+    // an operator logically equivalent to 'this' operator
+    // used by view based mv rewrite
+    // eg: LogicalViewScanOperator is logically equivalent to the operator build from the view
+    protected Operator equivalentOp;
```

Comment:
**logical equivalent** has same meaning as group expression, why not reuse the concept of group expression ?

Reply:
I think group expression is used in just memo optimization phase. But here we should decide it in RelationTransformer. And equivalentOp do not need cost/property and so on.

Reply:
should be `viewScanOperator`？I agree with @mofeiatwork , normal operator could use `equals` to check equivalent with another, `equivalentOp` is confusing